Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -109,8 +109,13 @@
     color : "color() * abs(${temperature} - 0.5)"
 });
 
-addStyle('Cos', {
-    color : "color() * cos(${temperature} - 0)"
+addStyle('Trigonometric Functions', {
+    color : "color() * radians(cos(${temperature} - 0)) + color() * sin(${temperature} - 0) + color() * tan(${temperature} - 0)"
+
+});
+
+addStyle('Arc Trigonometric Functions', {
+    color : "color() * degrees(acos(${temperature} - 0)) + color() * asin(${temperature} - 0) + color() * atan(${temperature} - 0)"
 });
```

Comment:
This test produces just a pure white, try tweaking the values a bit.

Also styling based on time is now possible with https://github.com/AnalyticalGraphicsInc/cesium/pull/4626 so you can incorporate that in some way.